docs/docs/theme-ui.md

```diff
@@ -29,6 +29,56 @@ module.exports = {
 Theme UI uses a `theme` configuration object to provide color, typography, layout, and other shared stylistic values through [React context][].
 This allows components within your site to add styles based on a predefined set of values.
 
+There are two ways to add a theme to your site.
+
+### `gatsby-plugin-theme-ui` configuration
+
+`gatsby-plugin-theme-ui` allows for optional configuration. One of the options you can pass in is `preset`. Preset is a base theme object you'd like your site to use.
+
+The `preset` can be a JSON object or a string package name that the plugin will require for you.
+
+If you're passing in a string package name you need to make sure that the dependency is installed.
+
+```shell
+npm install @theme-ui/preset-funk
+```
+
+```js:title=gatsby-config.js
+module.exports = {
+  plugins: [
+    {
+      resolve: `gatsby-plugin-theme-ui`,
+      options: {
+        preset: "@theme-ui/preset-funk",
+      },
+    },
+  ],
+}
+```
+
+Alternatively, you could do the same thing and require the package yourself.
+
+```shell
+npm install @theme-ui/preset-funk
+```
+
+```js:title=gatsby-config.js
+module.exports = {
+  plugins: [
+    {
+      resolve: `gatsby-plugin-theme-ui`,
+      options: {
+        preset: require("@theme-ui/preset-funk"),
+      },
+    },
+  ],
+}
+```
+
+`preset` operates as your base theme and any local shadowed styles with automatically merge with it. Local styles are considered more specific and will override base styles where necessary.
+
+### Local shadowing
+
 The Theme UI plugin uses the [component shadowing API][] to add the theme object context to your site.
```

Comment:
The link at the bottom of this doc is wrong, it should be: https://www.gatsbyjs.org/docs/theme-api/#shadowing

I also think it would help to quickly define shadowing in case people are super new to Themes.